Makefile.am

```diff
@@ -85,3 +86,9 @@ desktopdir = $(datadir)/applications
 desktop_in_files = geany.desktop
 desktop_DATA = $(desktop_in_files:.desktop.in=.desktop)
 @INTLTOOL_DESKTOP_RULE@
+
+appdata_in_files = geany.appdata.xml.in
+
+appdatadir = $(datadir)/appdata
```

Comment:
generally, there is a trend toward /usr/share/metainfo - for new code I'd recommend following this move.
See https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html#sect-Quickstart-DesktopApps